CMakeLists.txt

```diff
@@ -18,13 +18,25 @@ ELSEIF(${CMAKE_BUILD_TYPE} MATCHES "Debug")
   SET(CMAKE_CXX_FLAGS  "${CMAKE_CXX_FLAGS} -g")
 ENDIF()
 
+
 INCLUDE($ENV{ROS_ROOT}/core/rosbuild/rosbuild.cmake)
 ROSBUILD_INIT()
-SET(ROS_BUILD_SHARED_LIBS true)
 
+SET(ROS_BUILD_STATIC_LIBS true)
+SET(ROS_BUILD_SHARED_LIBS false)
+
+LIST(APPEND CMAKE_PREFIX_PATH "~/third_party/libtorch")
```

Comment:
The README.md instructions are to unpack libtorch to `/opt/libtorch`, so that path should be used instead.

Reply:
Sure; this is how it  needs to be on the jackal itself due to the filepaths there, but I can keep it as `opt` in the repo